functions.opengraph.php

```diff
@@ -349,10 +358,20 @@ function jetpack_og_get_image( $width = 200, $height = 200, $max_images = 4 ) {
 
 	// Third fall back, Core Site Icon, if valid in size. Added in WP 4.3.
 	if ( empty( $image ) && ( function_exists( 'has_site_icon') && has_site_icon() ) ) {
-		$image_url = get_site_icon_url( max( $width, $height ) );
-		$image_size = getimagesize( $image_url );
-		$img_width = $image_size[0];
-		$img_height = $image_size[1];
+		$max_side = max( $width, $height );
+		$image_url = get_site_icon_url( $max_side );
+
+		$img_width  = '';
+		$img_height = '';
+		$image_id = attachment_url_to_postid( $image_url );
```

Comment:
@eliorivero Instead of doing a reverse lookup using `attachment_url_to_postid` which can be very slow on big sites, can we just call `get_option( 'site_icon' )` to get the id?